subsys/metrics/src/main/conf/conf.d/metrics.conf

```diff
@@ -1,10 +1,21 @@
 [metrics]
 enabled = false
-reporter = graphite.reporter
+reporter = graphite.reporter;zabbix.reporter
```

Comment:
reporter.enable just use in ftest , metrics ftest just test metrics basic function , do not have reporter's function.